app/src/main/java/io/appium/uiautomator2/model/BaseElement.java

```diff
@@ -243,4 +248,28 @@ public boolean equals(Object other) {
                 && Objects.equals(contextId, otherEl.contextId)
                 && this.isSingleMatch == otherEl.isSingleMatch;
     }
+
+    /**
+     * Returns the string by combining all of keys/values in the given
+     * AccessibilityNodeInfo's extras. Separator is ";".
+     * "extras" is https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#getExtras()
+     *
+     * @param nodeInfo An AccessibilityNodeInfo instance
+     * @return the string of extra bundles, or null
+     */
+    @Nullable
+    public String getExtrasAsString(AccessibilityNodeInfo nodeInfo) {
```

Comment:
should this be a static helper instead? I don't see it is using any properties of this class